Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM golang:alpine
 
 MAINTAINER LinkedIn Burrow "https://github.com/linkedin/Burrow"
 
-RUN apk add --update bash curl git && rm -rf /var/cache/apk/*
+RUN apk add --update bash curl git && apk add ca-certificates wget && update-ca-certificatesrm -rf /var/cache/apk/*
```

Comment:
Contains a typo now, should end with `&& update-ca-certificates && rm -rf /var/cache/apk/*`